pennylane/transforms/control.py

```diff
@@ -188,7 +188,8 @@ def ctrl(fn, control, control_values=None):
     Args:
         fn (function): Any python function that applies pennylane operations.
         control (Wires): The control wire(s).
-        control_values (list[int]): The values the control wire(s) should take.
+        control_values (int or list[int]): The value(s) the control wire(s) should take.
+            Integers other than x=0 or x=1 will treated as int(bool(x)).

```

Comment:
```suggestion
            Integers other than 0 or 1 will treated as ``int(bool(x))``.
```
I don't think we need the `x=` part.

Reply:
Well it was to provide a name for argument in the `int(bool(...))` part but we can leave it out.